src/parser/ast/statement.rs

```diff
@@ -0,0 +1,61 @@
+use std::fmt::Debug;
+
+use super::{expression::Expression, DebugSymRef};
+
+#[derive(Clone)]
+pub enum Statement<F, V> {
+    AssertEq(DebugSymRef, Expression<F, V>, Expression<F, V>),
+    AssertNEq(DebugSymRef, Expression<F, V>, Expression<F, V>),
```

Comment:
why not assert of logical expression?

This way we can do:
```
assert is_foo and is_bar
```
or
```
assert is_foo or is_bar
```
Then an eq is an expression that returns an logical

Reply:
Added generic assert statement